### DISCUSSION TITLE
No reference to CN in CAN - eForms format

### DISCUSSION THREAD

**Discussion by poetsmeniet** [OP-TED/eForms-SDK#820] [Q&A] [answered]
In the new eForms format:

I can't find a reference to a Contract Notice in the following Contract Award Notice: 51276-2024

It was pretty convenient to have this reference in the old format. Is this not part of eForms?

**Comment by YvesJo** [comment, marked as answer]
Hi,
Indeed, in eForms, Competition and Result notices will share a same Procedure ID (BT-04) and there is no other need to associate notices together.
KR
